azure-pipelines.yml

```diff
@@ -64,19 +83,23 @@ jobs:
 # Debug build
 #
 
-- template: eng/platform-matrix.yml
-  parameters:
-    jobTemplate: build-job.yml
-    buildConfig: debug
+# Official build is release only.
+- ${{ if and(ne(variables['System.TeamProject'], 'public'), notin(variables['Build.Reason'], 'PullRequest')) }}:
+  - template: eng/platform-matrix.yml
+    parameters:
+      jobTemplate: build-job.yml
+      buildConfig: debug
 
 #
 # Checked build
 #
 
-- template: eng/platform-matrix.yml
-  parameters:
-    jobTemplate: build-job.yml
-    buildConfig: checked
+# Official build is release only.
+- ${{ if and(ne(variables['System.TeamProject'], 'public'), notin(variables['Build.Reason'], 'PullRequest')) }}:
+  - template: eng/platform-matrix.yml
```

Comment:
You might be able to put all of the templates under the same ```if``` statement.

Reply:
I think I would prefer to have them separate, as the other way requires duplicating the expansion logic